packages/cli/src/cmds/validator/options.ts

```diff
@@ -297,7 +295,8 @@ export const validatorOptions: CliCommandOptions<IValidatorCliArgs> = {
 
   "externalSigner.fetch": {
     conflicts: ["externalSigner.pubkeys"],
-    description: "Fetch then list of public keys to validate from an external signer",
+    description:
+      "Fetch the list of public keys to validate from an external signer. Cannot be used in combination with `--externalSigner.pubkeys`",
```

Comment:
If both options are set, Lodestar will throw the following error on startup
```
✖ Arguments externalSigner.fetch and externalSigner.pubkeys are mutually exclusive
```
but there is no hint at this by just looking at the docs (although user could infer it from the behavior)